vcenter_operator/phelm.py

```diff
@@ -77,18 +77,24 @@ def _apply_item(self, resource, resource_args, new_item):
         client = self.get_client()
         metadata_name = new_item['metadata']['name']
 
-        # If anything has changed, the server will trigger it
-        client.server_side_apply(resource, new_item,
-                                 force_conflicts=True,  # Sole controller
-                                 **resource_args)
         if self.dry_run:
-            LOG.info(f"Apply: {resource}/{metadata_name}")
+            LOG.info(f"Applying: {resource}/{metadata_name}")
             for line in json.dumps(
                     new_item, sort_keys=True,
                     indent=2, separators=(',', ': ')).splitlines():
                 LOG.debug(line)
         else:
-            LOG.debug(f"Apply: {resource}/{metadata_name}")
+            LOG.debug(f"Applying: {resource}/{metadata_name}")
+
+        # If anything has changed, the server will trigger it
+        try:
+            client.server_side_apply(resource, new_item,
+                                    force_conflicts=True,  # Sole controller
+                                    **resource_args)
+        except dynamic.exceptions.UnprocessibleEntityError:
+            # If the server can't patch it, try to replace it
```

Comment:
Maybe add a log line here? Or does it happen so regularly that it'd be spammy?

Reply:
No, it should happen only very rarely. Will do.